tests/e2e/performance/test_pvc_creation_performance.py

```diff
@@ -11,13 +11,24 @@
 from ocs_ci.framework.testlib import performance, E2ETest, polarion_id, bugzilla
 from ocs_ci.helpers import helpers
 from ocs_ci.ocs import defaults, constants
-
+from ocs_ci.utility.utils import exec_cmd
+from ocs_ci.ocs.resources.pvc import (
+    get_all_pvcs_in_storageclass,
+    get_all_pvs_in_storageclass,
+)
 
 from ocs_ci.utility.performance_dashboard import push_to_pvc_time_dashboard
+from ocs_ci.ocs.cluster import CephCluster
 
 log = logging.getLogger(__name__)
 
 
+def get_storage_pool_name(interface, namespace):
```

Comment:
@Avilir I used this function in my test execution and the function failed to get the pool_name, observed the problem is duet to the pool name.
In my test_case am not creating new SC, used the default CephFS/RBD SC in cluster belong to openshift-storage but during function call used the namespace created for PVC's which failed to get pool_name.
Based on above I would recommend to update param `namespace = "openshift-storage"`

Reply:
Please add doc string for this function.

Reply:
done